server/build_event_protocol/target_tracker/BUILD

```diff
@@ -54,15 +54,16 @@ go_test(
         "--app.enable_write_test_target_statuses_to_olap_db",
     ],
     exec_properties = {
-        "test.workload-isolation-type": "firecracker",
-        "test.init-dockerd": "true",
         "test.recycle-runner": "true",
         # We don't want different different db tests to be assigned to the samed
         # recycled runner, because we can't fit all db docker images with the
         # default disk limit.
         "test.runner-recycling-key": "clickhouse",
     },
     tags = ["docker"],
+    target_compatible_with = [
+        "//platforms/configs:with_docker_daemon",
+    ],
```

Comment:
I'd rather not make this incompatible locally until we have proper `dockerd` detection, because if folks have a dockerd running locally then they can test this by running with `bazel test //... --test_tag_filters=+docker` but IIUC this change would make it so that the target cannot be run locally at all. (Same comment for the other targets)